sandbox/sandbox/src/host_executor.rs

```diff
@@ -85,7 +142,10 @@ impl super::SandboxMemory for Memory {
         }
     }
 
-    fn get(&self, offset: u32, buf: &mut [u8]) -> Result<(), Error> {
+    fn get<C>(&self, _ctx: &C, offset: u32, buf: &mut [u8]) -> Result<(), Error>
```

Comment:
don't we want to make fn name more informative like get -> get_mem?

Reply:
`SandboxMemory::get_mem` seems strange

Reply:
but get what
get_buff a few lines below is enough intuitive

anyway, not the deal for me, just paying your attention

Reply:
Yes, this is default substrate naming. I think better read_to_buf()